packages/terra-date-picker/src/DatePicker.jsx

```diff
@@ -239,6 +248,7 @@ class DatePicker extends React.Component {
       onClickOutside,
       onSelect,
       requestFocus,
+      required,
```

Comment:
Should add `aria-required`. See [form-input](https://github.com/cerner/terra-core/blob/master/packages/terra-form-input/src/Input.jsx#L109-L111 ).

Reply:
Thanks for catching that: https://github.com/cerner/terra-framework/pull/504/commits/85f234d80bd58813cc7d4a3f3cdd119875811db6